clusters/app.ci/supplemental-ci-images/baremetal-qe/coreos-installer.yaml

```diff
@@ -0,0 +1,15 @@
+kind: ImageStream
+apiVersion: image.openshift.io/v1
+metadata:
+  namespace: ci
+  name: coreos-installer
+spec:
+  tags:
+  - name: "release"
+    from:
+      kind: DockerImage
+      name: quay.io/coreos/coreos-installer:release
+    importPolicy:
+      scheduled: false
```

Comment:
for future purposes, may you add the following within the importPolicy stanza?
```
importMode: PreserveOriginal
```

it will allow manifest-list images to be imported correctly

cc @droslean to double check and confirm

Reply:
thanks, added